context/app/markdown/docs/assay/seqfish.md

```diff
@@ -15,6 +15,7 @@ seqFISH technology allows in situ imaging of multiple mRNAs using barcoding and
 *Figure 1:* Each mRNA is barcoded with a combination of 3 barcode regions, numbered I-III (region IV is used for error correction). In turn, each barcode region is decoded by hybridization of 1 of 20 possible readout barcodes per channel (203 = 8000 unique barcodes per channel) X 3 channels (24,000 unique barcodes). The complete barcode for each mRNA is deduced from the sequence of readout barcodes for regions I-III.
 
 ![](https://lh3.googleusercontent.com/oHCVhh_9ZeoqRhEcflRjARPAdtxRCs1-lC2W4X_WNX-fBFYDC0tbkL2hDrrXVvvUGc0BVF-0eaYoi4jr0XRNz_QbcrDB4UWQjSnIHQNrtxkS97ZBnKQkkfYilBaGxOFzz6CFh-s)
```

Comment:
@schwenk102 @pecan88 I have a content concern: this image is a screenshot that someone grabbed from a Google search. We can't just make that part of our documentation, it could get us into legal trouble.

Reply:
That's not a change in this PR, but I've filed #599 where it can be followed up. I'll go ahead and merge this.

Reply:
Here's another with the citation: https://www.researchgate.net/publication/322568660_Visual_perception_of_the_osseous_labyrinth_rendered_from_micro-CT_scans_of_the_petrous_bone
![Screen Shot 2020-07-02 at 5 24 20 PM](https://user-images.githubusercontent.com/8284878/86410717-4b69fc80-bc89-11ea-84e9-b6cfb1c32f23.png)